### PR TITLE
Fix #18696: Construction rights cannot be viewed after all are purchased

### LINKED ISSUE #18696: Construction rights cannot be viewed after all are purchased
### Operating System

Windows 10 64 bit

### OpenRCT2 build

OpenRCT2, v0.4.2 (8ceea45) provided by GitHub

### Base game

RollerCoaster Tycoon 2

### Describe the issue

Once all construction rights have been purchased, there is no longer any way to actually view the tiles you have construction rights for, because the construction rights button gets disabled.  There should be an additional "view construction rights" button in the land rights dialog to view purchased construction rights.  On a map like Urban Park, for instance, it becomes very tricky to build when you bought all the construction rights but can't see which tiles you're actually allowed to construct over/under.

### Steps to reproduce

1. Open Land Rights dialog.
2. Purchase all construction rights.
3. Close and open Land Rights dialog again.  Construction rights cannot be viewed.

### Attachments

_No response_

### PR DESCRIPTION
**Pull request by rik-smeets**
Resolves #18696.

### CODE REVIEW

File: src/openrct2/interface/Viewport.cpp [hunk 1]
```diff
@@ -45,7 +45,7 @@
 
 using namespace OpenRCT2;
 
-//#define DEBUG_SHOW_DIRTY_BOX
+// #define DEBUG_SHOW_DIRTY_BOX
```
Comment (Broxzier) [inline src/openrct2/interface/Viewport.cpp:48]: I guess this is to satisfy clang-format? If so, strange that it only comes up now.

Comment (rik-smeets) [reply src/openrct2/interface/Viewport.cpp:48]: Yes, it's because of clang-format. I've seen this before, don't know why they don't show up earlier.

Comment (Gymnasiast) [reply src/openrct2/interface/Viewport.cpp:48]: Let’s get rid of it altogether (in a separate commit), we shouldn’t have commented-out code anyway.

Comment (rik-smeets) [reply src/openrct2/interface/Viewport.cpp]: Good point, I deleted it. I also fixed a typo which I only just noticed in the global field.

**Review by Gymnasiast — APPROVED** [review]

**Pull request merged**